### PR TITLE
Minor tweaks to Enterprise docs

### PR DESCRIPTION
**Pull request by rcgenova**
-Rewords introductory paragraph
-Slight wording changes elsewhere
-Adds additional links to the Enterprise product page

### CODE REVIEW

File: website/source/docs/enterprise/index.html.md [hunk 1]
```diff
@@ -3,18 +3,20 @@ layout: "docs"
 page_title: "Nomad Enterprise"
 sidebar_current: "docs-enterprise"
 description: |-
-  Nomad Enterprise features a number of capabilities beyond the open source
-  offering that may be beneficial in certain workflows.
+  Nomad Enterprise adds collaboration and governance capabilities to Nomad.
+  Features include Namespaces, Resource Quotas and Sentinel Policies.
```
Comment (preetapan) [inline website/source/docs/enterprise/index.html.md]: What is our stance on the Oxford Comma? I am usually for it. Consul team also enforced it in our docs. I would vote to keep things consistent.

Comment (rcgenova) [reply website/source/docs/enterprise/index.html.md]: That works for me. I added the one after "Resource Quotas" above. Let me know if have missed any others.

File: website/source/docs/enterprise/sentinel/index.html.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ description: |-
 # Nomad Enterprise Sentinel Policy Enforcement
 
 In [Nomad Enterprise](https://www.hashicorp.com/products/nomad/), operators can
-create [Sentinel policies](/guides/sentinel-policy.html) for fine grain policy
-enforcement.  Sentinel policies build on top of the ACL system and allow operators to define
-fine grain policies such as disallowing jobs to be submitted to production on
+create [Sentinel policies](/guides/sentinel-policy.html) for fine-grained policy
+enforcement. Sentinel policies build on top of the ACL system and allow operators to define
```
Comment (preetapan) [inline website/source/docs/enterprise/sentinel/index.html.md]: comma after system

File: website/source/guides/sentinel-policy.html.markdown [hunk 3]
```diff
@@ -3,12 +3,12 @@ layout: "guides"
 page_title: "Sentinel Policies"
 sidebar_current: "guides-sentinel"
 description: |-
- Nomad integrates with Sentinel for fine grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the ACL System.
+ Nomad integrates with Sentinel for fine-grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the ACL System.
```
Comment (preetapan) [inline website/source/guides/sentinel-policy.html.markdown]: comma after sandbox

File: website/source/guides/sentinel-policy.html.markdown [hunk 4]
```diff
@@ -3,12 +3,12 @@ layout: "guides"
...
 ---
 
 # Sentinel Policies
 
-Nomad integrates with [HashiCorp Sentinel](https://docs.hashicorp.com/sentinel) for fine grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the [ACL System](/guides/acl.html).
+[Nomad Enterprise](https://www.hashicorp.com/products/nomad/) integrates with [HashiCorp Sentinel](https://docs.hashicorp.com/sentinel) for fine-grained policy enforcement. Sentinel allows operators to express their policies as code, and have their policies automatically enforced. This allows operators to define a "sandbox" and restrict actions to only those compliant with policy. The Sentinel integration builds on the [ACL System](/guides/acl.html).
```
Comment (preetapan) [inline website/source/guides/sentinel-policy.html.markdown]: comma after sandbox

File: website/source/docs/enterprise/sentinel/index.html.md (hunk 2 above)
Comment (rcgenova) [reply website/source/docs/enterprise/sentinel/index.html.md]: I believe the Oxford comma rules apply to a series of three or more items: https://en.wikipedia.org/wiki/Serial_comma

Comment (preetapan) [reply website/source/docs/enterprise/sentinel/index.html.md]: Ah, thanks for clarifying.

**Pull request merged**